src/main/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperations.scala

```diff
@@ -763,6 +763,19 @@ object TezosDatabaseOperations extends LazyLogging {
   def writeProcessedEventsLevels(eventType: String, levels: List[BigDecimal]): DBIO[Option[Int]] =
     Tables.ProcessedChainEvents ++= levels.map(Tables.ProcessedChainEventsRow(_, eventType))
 
+  /** Adds any new level for which a custom event processing has been executed
+    *
+    * @param rows the type of event to record
+    * @return the number of entries saved added to the
+    */
+  def writeRegisteredTokensRowsIfEmpty(
+      rows: List[Tables.RegisteredTokensRow]
+  )(implicit ec: ExecutionContext): DBIO[Option[Int]] =
```

Comment:
Personally I'd express the return value more explicitly to understand what happened, to avoid confusion.

Right now, does an empty list of rows to save look exactly as if the db table wasn't empty?

Also, at call site (in Lorre), I feel the message handling the `None` return value is incorrect, as it's returned only if the database driver will not be able to provide the added row count, even when succeeding.

Reply:
I made simple changes, but can you elaborate on how do you think it should look like?